app/mailers/views/stem_applicant_confirmation.html.erb

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta content='text/html; charset=UTF-8' http-equiv='Content-Type' />
+  </head>
+  <body>
+    <p>Dear <%= first_and_last_name %>,</p>
+
+    <p>VA received your application for the Rogers STEM scholarship on <%= application_date %>.</p>
+
+    <p>We usually process applications within 30 days. We may contact you if we need more information or documents.</p>
+
+    <p>
+      <b>Rogers STEM Scholarship (Form 22-10203)</b>
+      <br>
+      for <%= first_and_last_name %>
+      <br>
+      Confirmation number <%= @claim.education_benefits_claim.confirmation_number %>
+      <br>
+      Date received <%= application_date %>
+    </p>
+
+    <p>
+      <b>Your claim was sent to:</b>
+      <br />
+      Eastern Region
+      <br />
+      VA Regional Office
+      <br />
+      P.O. Box 4616
+      <br />
+      Buffalo, NY 14240-4616
```

Comment:
Is this handy in the data? I don't think it will change, but theoretically it could. If it is handy, it would be nice if it was dynamic, otherwise not necessary.

Reply:
Yeah, I think the address is a constant value so I'll try to pull that in